src/App.scss

```diff
@@ -7,4 +7,45 @@ body {
   box-sizing: border-box;
   margin: 0 auto;
   font-family: 'Roboto';
+  font-size: 20px;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    font-size: 10px;
+  }
+  .beer-card {
+    flex-direction: column;
+  }
+
+  .img {
+    width: 90%;
+    height: 300px;
+  }
+  .search-page {
+    margin-left: 20px;
+    width: 95%;
+  }
+  .favourite-beer {
+    margin-left: 20px;
+  }
+
+  .beers-per-page {
+    align-self: flex-start;
+  }
+  .navigation-page-buttons {
+    width: 450px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  .button-find-beers {
+    align-self: flex-start;
+    margin-top: 20px;
+  }
+  .random-beer {
+    margin-left: 20px;
+  }
+  .cart {
```

Comment:
Is this typo (card)?

Reply:
Yes but it's not good name this class. I change to 'icon-position'